src/pages/guidelines/spacing.mdx

```diff
@@ -76,85 +72,166 @@ Below is the example of a regular Content block simple on the left and how the s
-| `$spacing-06` | 1.5   | 24  | <div style={{width: '24px', height: '24px', background: '#D12771'}}></div>   |
-| `$spacing-07` | 2     | 32  | <div style={{width: '32px', height: '32px', background: '#D12771'}}></div>   |
-| `$spacing-08` | 2.5   | 40  | <div style={{width: '40px', height: '40px', background: '#D12771'}}></div>   |
-| `$spacing-09` | 3     | 48  | <div style={{width: '48px', height: '48px', background: '#D12771'}}></div>   |
-| `$spacing-10` | 4     | 64  | <div style={{width: '64px', height: '64px', background: '#D12771'}}></div>   |
-| `$spacing-11` | 5     | 80  | <div style={{width: '80px', height: '80px', background: '#D12771'}}></div>   |
-| `$spacing-12` | 6     | 96  | <div style={{width: '96px', height: '96px', background: '#D12771'}}></div>   |
-| `$spacing-13` | 10    | 160 | <div style={{width: '160px', height: '160px', background: '#D12771'}}></div> |
+| `$spacing-01` | 0.125 | 2   | <div style={{width: '2px', height: '2px', background: '#A56EFF'}}></div>     |
+| `$spacing-02` | 0.25  | 4   | <div style={{width: '4px', height: '4px', background: '#A56EFF'}}></div>     |
+| `$spacing-03` | 0.5   | 8   | <div style={{width: '8px', height: '8px', background: '#A56EFF'}}></div>     |
+| `$spacing-04` | 0.75  | 12  | <div style={{width: '12px', height: '12px', background: '#A56EFF'}}></div>   |
+| `$spacing-05` | 1     | 16  | <div style={{width: '16px', height: '16px', background: '#A56EFF'}}></div>   |
+| `$spacing-06` | 1.5   | 24  | <div style={{width: '24px', height: '24px', background: '#A56EFF'}}></div>   |
+| `$spacing-07` | 2     | 32  | <div style={{width: '32px', height: '32px', background: '#A56EFF'}}></div>   |
+| `$spacing-08` | 2.5   | 40  | <div style={{width: '40px', height: '40px', background: '#A56EFF'}}></div>   |
+| `$spacing-09` | 3     | 48  | <div style={{width: '48px', height: '48px', background: '#A56EFF'}}></div>   |
+| `$spacing-10` | 4     | 64  | <div style={{width: '64px', height: '64px', background: '#A56EFF'}}></div>   |
+| `$spacing-11` | 5     | 80  | <div style={{width: '80px', height: '80px', background: '#A56EFF'}}></div>   |
+| `$spacing-12` | 6     | 96  | <div style={{width: '96px', height: '96px', background: '#A56EFF'}}></div>   |
+| `$spacing-13` | 10    | 160 | <div style={{width: '160px', height: '160px', background: '#A56EFF'}}></div> |
 
 </div>
 
 <br />
 
-## Content component spacing patterns
+## Spacing concepts
+
+### Internal vs external spacing
+
+Components containing multiple elements may have internal spacing to help guide how those elements display as the component resizes across breakpoints.
+
+#### Internal spacing
+
+The internal spacing for card determines how the various elements are spaced within the container, including when some elements are visible or hidden.
+
+<Row>
+<Column colLg={8}>
+
+![A card component with internal spacing highlighted](../../images/guidelines/spacing/guidelines-spacing-spacing-concepts-01.png)
```

Comment:
Not a huge deal, but is the arrow purple on the card here? If it's an old image we don't have to update